.github/workflows/ci.yaml

```diff
@@ -32,56 +32,61 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       tags: ${{ steps.meta.outputs.tags }}
+
     steps:
     - name: Checkout
       uses: actions/checkout@v2
 
-    # Generates tags https://github.com/crazy-max/ghaction-docker-meta
-    - name: Docker meta
+    - name: Generate Docker Tags
       id: meta
-      uses: crazy-max/ghaction-docker-meta@v3.1.0
+      uses: docker/metadata-action@v3
       with:
-        images: ${{ env.DOCKERHUB_REPO }}
+        images: ${{ env.DOCKER_REPO }}
         tags: |
           type=ref,event=pr,priority=30
           type=semver,pattern=v{{version}},priority=20
           type=sha,priority=10
+      # Priority matters because the chart deploy should use a specific tag over "latest".
 
     - name: Set up Docker Buildx
+      id: buildx
       uses: docker/setup-buildx-action@v1
-
-    - name: Cache Docker layers
-      uses: actions/cache@v2
       with:
-        path: /tmp/.buildx-cache
-        key: ${{ runner.os }}-buildx-${{ github.sha }}
-        restore-keys: |
-          ${{ runner.os }}-buildx-
+        install: true
 
     - name: Login to DockerHub
       uses: docker/login-action@v1
       with:
         username: ${{ secrets.DOCKERHUB_USERNAME }}
         password: ${{ secrets.DOCKERHUB_TOKEN }}
 
-    - name: Build and push
+    - name: Build
       id: docker_build
       uses: docker/build-push-action@v2
       with:
         context: .
-        push: true
+        load: true
         tags: ${{ steps.meta.outputs.tags }}
-        labels: ${{ steps.meta.outputs.labels }}
-        cache-from: type=local,src=/tmp/.buildx-cache
-        cache-to: type=local,dest=/tmp/.buildx-cache-new
+        cache-from: type=gha
+        cache-to: type=gha,mode=max
```

Comment:
Just so I understand: do the simplifications here allow us to remove the explicit cache step above at https://github.com/mobilecoinofficial/mobot/pull/75/commits/28fbcd9ed50c1543c9baf83653960c2ff1f33973#diff-944291df2c9c06359d37cc8833d182d705c9e8c3108e7cfe132d61a06e9133ddL53-L59?